tests/checkup_test.go

```diff
@@ -307,6 +310,15 @@ func newConfigMap() *corev1.ConfigMap {
 	}
 }
 
+func getJobTimeout(configmapData map[string]string) (time.Duration, error) {
+	const bufferTime = 5 * time.Minute
+	testTimeout, err := time.ParseDuration(configmapData["spec.timeout"])
```

Comment:
Please consider adding a variable to hold the timeout value we set in the ConfigMap.
By doing that, parsing the timeout from the ConfigMap becomes redundant.

Reply:
It would greatly reduce the complexity of the PR, but at the cost of dynamicality.. 
I tend to agree it's better to keep things simple.